addons/Lookup/README.md

```diff
@@ -0,0 +1,100 @@
+# Lookup
+A simple [Windower4](http://www.windower.net/) addon that looks up search terms through in-game commands.
+
+The default search is performed with the following command:
+```
+//lookup "Search Term"
+```
+This will open up the search in your default browser.
+
+The search term can be plain text, auto-translate text, or one of the available [selectors](#selectors). If the search term contains a space, it must be surrounded in quotes (this does not apply to selectors).
+
+The available search sites are [FFXIclopedia](http://ffxiclopedia.wikia.com/), [BGWiki](http://www.bg-wiki.com), [FFXIAH](http://www.ffxiah.com), [FFXIDB](http://www.ffxidb.com), and [Google](http://www.google.com).
+
+See the [commands](#commands) section for a list of all available commands.
+
+## Selectors
+Selectors can be used in place of plain text search terms. They are very useful for quickly getting information about something in the environment or a recently obtained item.
+
+| Selector | Replacement |
+|----------|-------------|
+| `<t>` | The current target's name. |
+| `<bt>` | The current battle target'name . |
+| `<pet>` | The name of the current player's pet. |
+| `<me>` | The current player's name. |
+| `<r>` | The name of the player that last sent a tell to you. |
+| `<job>` | The current player's main job. |
+| `<subjob>`<br>`<sj>` | The current player's subjob. |
+| `<area>`<br>`<zone>` | The current area/zone. |
+| `<item>`<br>`<lastitem>` | The last item placed in the player's inventory, including items moved from other bags. |
+
```

Comment:
Since this uses `windower.ffxi.get_mob_by_target` it actually supports a lot more than these, although I'm not sure if it's worth it to mention... technically all immediate target identifiers that FFXI allows in `<>` can also be used here, like `<pX>`, `<aX>`, `<ht>` and so on. But again, not sure the use case is big enough to warrant mentioning here...

Reply:
Yeah, I picked a few that would be most useful, since the total list looked too cluttered. Maybe I should add a link to the rest, though.